src/Buttons.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import './Buttons.css';
+import App from "./App";
+
+function Buttons(){
+    return(
+
+        <div className="keys">
+            <div class={"container"}>
+                    <div>
```

Comment:
Fix indentation here and remove unnecessary white space within the return() block